game/utils/keycodec.cpp

```diff
@@ -337,6 +339,14 @@ void KeyCodec::keyToStr(Tempest::Event::MouseButton k, char* buf, size_t bufSz)
     std::strncpy(buf,"MOUSE RIGHT",bufSz);
     return;
     }
+  if(k==Tempest::Event::ButtonForward) {
+    std::strncpy(buf,"MOUSE 5",bufSz);
+    return;
+    }
+  if(k==Tempest::Event::ButtonBack) {
+    std::strncpy(buf,"MOUSE 4",bufSz);
```

Comment:
In steam version of the game this key is `MOUSE X1` and `MOUSE X2` and code is `0x0f02`,`0x1002` (tested with UNION installed)
We should aligned to it